waterlog/package.json

```diff
@@ -23,7 +23,6 @@
     "react-chartjs-2": "^2.7.4",
     "react-dom": "^16.7.0",
     "react-leaflet": "^2.2.0",
-
```

Comment:
Why is this file being committed?

Reply:
I didn't know that it isn't supposed to be committed, shouldn't it be?

Reply:
No it shouldn't

Reply:
i didn't know that it is not supposed to be committed. Should i remove it?